app/controllers/time_tracker_controller.rb

```diff
@@ -85,7 +85,7 @@ def default_end_time_for_timer(timer_session)
   end
 
   def set_timer_session
-    @timer_session = TimerSession.active_session(@current_user.id).first
+    @timer_session = TimerSession.active_sessions.find_by(user: @current_user)
```

Comment:
Some questions:
* is it safe to assume that there's always only one?
* don't we have a `current_user` method instead of using the instance variable directly?
* the same logic is repeated below, why not having a `current_timer_session` method?